intellij/src/main/resources/META-INF/plugin.xml

```diff
@@ -1,72 +1,18 @@
 <idea-plugin version="2">
     <id>com.gluonhq.plugin.intellij</id>
     <name>Gluon Plugin</name>
-    <version>2.6.2</version>
+    <version>2.7.0</version>
     <vendor email="support@gluonhq.com" url="http://gluonhq.com">Gluon</vendor>
 
     <description><![CDATA[
       The Gluon Plugin allows developers to create Java applications that target the Android and iOS mobile platforms as well as the Desktop platform using the same codebase.
     ]]></description>
 
     <change-notes><![CDATA[
-      <b>2.6.3</b>
+      <b>2.7.0</b>
       <ul>
-      <li>Introduce Single View project using Gluon VM</li>
+      <li>Update templates for Gluon Mobile 5</li>
       </ul>
-      <b>2.6.2</b>
```

Comment:
Are older release notes stored somewhere?

Reply:
Yes, we added a CHANGELOG.md file for each plugin, inside the repository itself.

These notes are only used as information on the IntelliJ plugin page. On that page, the release notes are separated per version, so it doesn't make sense to include the entire release notes each time for each release. For example, this is the page for version 2.6.3 https://plugins.jetbrains.com/plugin/7864-gluon-plugin/update/42325.

Reply:
Yes, it stored in a [changelog file](https://github.com/gluonhq/ide-plugins/blob/master/intellij/CHANGELOG.md).